<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCubeSeedFinder</class>
 <widget class="QWidget" name="GameCubeSeedFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameCube Seed Finder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabGales">
      <attribute name="title">
       <string>Gales</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGalesSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Search</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGalesRound">
            <property name="text">
             <string>Round #1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGalesResults">
            <property name="text">
             <string>Possible Results:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonGalesSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonGalesCancel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButtonGalesReset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGalesPokemon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Pok√©mon</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="3" column="1">
           <widget class="QRadioButton" name="radioButtonGalesMoltres">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesEnemyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QRadioButton" name="radioButtonGalesKangaskhan">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesEnemyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButtonGalesMew">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesYourLead</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButtonGalesArticuno">
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesEnemyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QRadioButton" name="radioButtonGalesLatias">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesEnemyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="radioButtonGalesJirachi">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesYourLead</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelGalesEnemyLead">
            <property name="text">
             <string>Enemy Lead</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButtonGalesDeoxys">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesYourLead</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonGalesMewtwo">
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesYourLead</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioButtonGalesZapdos">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesEnemyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="radioButtonGalesRayquaza">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupGalesYourLead</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelGalesYourLead">
            <property name="text">
             <string>Your lead</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxGalesHPStat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>HP Stat</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="2" column="0">
           <widget class="TextBox" name="textBoxGalesTopLeft"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGalesTopLeft">
            <property name="text">
             <string>Top Left</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGalesBottomLeft">
            <property name="text">
             <string>Bottom Left</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelGalesBottomRight">
            <property name="text">
             <string>Bottom Right</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="TextBox" name="textBoxGalesBottomLeft"/>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxGalesTopRight"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelGalesTopRight">
            <property name="text">
             <string>Top Right</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="TextBox" name="textBoxGalesBottomRight"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="labelGalesInfo">
            <property name="text">
             <string>Enter starting from Round 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBarGales">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColo">
      <attribute name="title">
       <string>Colo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxColoTrainer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Trainer Name</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButtonColoWes">
            <property name="text">
             <string>Wes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoTrainerName</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonColoSeth">
            <property name="text">
             <string>Seth</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoTrainerName</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButtonColoThomas">
            <property name="text">
             <string>Thomas</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoTrainerName</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxColoSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Search</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonColoSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelColoResults">
            <property name="text">
             <string>Possible Results:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButtonColoReset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonColoCancel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelColoRound">
            <property name="text">
             <string>Round #1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxColoParty">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Party Lead</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButtonColoSuicune">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="radioButtonColoHeracross">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButtonColoRaikou">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButtonColoBlaziken">
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButtonColoMeganium">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButtonColoEntei">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioButtonColoMetagross">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButtonColoSwampert">
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupColoPartyLead</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBarColo">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabChannel">
      <attribute name="title">
       <string>Channel</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxChannelSearch">
         <property name="title">
          <string>Search</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <widget class="QLabel" name="labelChannelResult">
            <property name="text">
             <string>Result:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonChannelSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonChannelCancel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxInputs">
         <property name="title">
          <string>Inputs</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textEditInputs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxChannelSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButtonSNOC">
              <property name="text">
               <string>S-&gt;N-&gt;O-&gt;C</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="radioButtonNSOC">
              <property name="text">
               <string>N-&gt;S-&gt;O-&gt;C</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButtonONSC">
              <property name="text">
               <string>O-&gt;N-&gt;S-&gt;C</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="radioButtonNOSC">
              <property name="text">
               <string>N-&gt;O-&gt;S-&gt;C</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButtonSNCO">
              <property name="text">
               <string>S-&gt;N-&gt;C-&gt;O</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="radioButtonNSCO">
              <property name="text">
               <string>N-&gt;S-&gt;C-&gt;O</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="radioButtonCNSO">
              <property name="text">
               <string>C-&gt;N-&gt;S-&gt;O</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QRadioButton" name="radioButtonNCSO">
              <property name="text">
               <string>N-&gt;C-&gt;S-&gt;O</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="radioButtonONCS">
              <property name="text">
               <string>O-&gt;N-&gt;C-&gt;S</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="radioButtonNOCS">
              <property name="text">
               <string>N-&gt;O-&gt;C-&gt;S</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QRadioButton" name="radioButtonCNOS">
              <property name="text">
               <string>C-&gt;N-&gt;O-&gt;S</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QRadioButton" name="radioButtonNCOS">
              <property name="text">
               <string>N-&gt;C-&gt;O-&gt;S</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupChannel</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonChannelAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonChannelRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBarChannel">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButtonGalesSearch</tabstop>
  <tabstop>pushButtonGalesCancel</tabstop>
  <tabstop>pushButtonGalesReset</tabstop>
  <tabstop>radioButtonGalesMewtwo</tabstop>
  <tabstop>radioButtonGalesMew</tabstop>
  <tabstop>radioButtonGalesDeoxys</tabstop>
  <tabstop>radioButtonGalesRayquaza</tabstop>
  <tabstop>radioButtonGalesJirachi</tabstop>
  <tabstop>radioButtonGalesArticuno</tabstop>
  <tabstop>radioButtonGalesZapdos</tabstop>
  <tabstop>radioButtonGalesMoltres</tabstop>
  <tabstop>radioButtonGalesKangaskhan</tabstop>
  <tabstop>radioButtonGalesLatias</tabstop>
  <tabstop>textBoxGalesTopLeft</tabstop>
  <tabstop>textBoxGalesTopRight</tabstop>
  <tabstop>textBoxGalesBottomLeft</tabstop>
  <tabstop>textBoxGalesBottomRight</tabstop>
  <tabstop>pushButtonColoSearch</tabstop>
  <tabstop>pushButtonColoCancel</tabstop>
  <tabstop>pushButtonColoReset</tabstop>
  <tabstop>radioButtonColoWes</tabstop>
  <tabstop>radioButtonColoSeth</tabstop>
  <tabstop>radioButtonColoThomas</tabstop>
  <tabstop>radioButtonColoBlaziken</tabstop>
  <tabstop>radioButtonColoEntei</tabstop>
  <tabstop>radioButtonColoSwampert</tabstop>
  <tabstop>radioButtonColoRaikou</tabstop>
  <tabstop>radioButtonColoMeganium</tabstop>
  <tabstop>radioButtonColoSuicune</tabstop>
  <tabstop>radioButtonColoMetagross</tabstop>
  <tabstop>radioButtonColoHeracross</tabstop>
  <tabstop>pushButtonChannelSearch</tabstop>
  <tabstop>pushButtonChannelCancel</tabstop>
  <tabstop>radioButtonSNOC</tabstop>
  <tabstop>radioButtonONSC</tabstop>
  <tabstop>radioButtonSNCO</tabstop>
  <tabstop>radioButtonCNSO</tabstop>
  <tabstop>radioButtonONCS</tabstop>
  <tabstop>radioButtonCNOS</tabstop>
  <tabstop>pushButtonChannelAdd</tabstop>
  <tabstop>radioButtonNSOC</tabstop>
  <tabstop>radioButtonNOSC</tabstop>
  <tabstop>radioButtonNSCO</tabstop>
  <tabstop>radioButtonNCSO</tabstop>
  <tabstop>radioButtonNOCS</tabstop>
  <tabstop>radioButtonNCOS</tabstop>
  <tabstop>pushButtonChannelRemove</tabstop>
  <tabstop>textEditInputs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupColoTrainerName"/>
  <buttongroup name="buttonGroupChannel"/>
  <buttongroup name="buttonGroupGalesYourLead"/>
  <buttongroup name="buttonGroupColoPartyLead"/>
  <buttongroup name="buttonGroupGalesEnemyLead"/>
 </buttongroups>
</ui>
